modnet/featurizers/utils.py

```diff
@@ -15,8 +15,8 @@ def clean_df(df):
 
     """
 
-    df = df.dropna(axis=1, how='all')
+    df = df.dropna(axis=1, how="all")
     df = df.replace([np.inf, -np.inf, np.nan], -1)
```

Comment:
for a future update on MODNet: should think on what is the most appropriate way to deal with nans and inf. For now, we can leave it like that of course